packages/commonwealth/client/scripts/identifiers.ts

```diff
@@ -24,10 +24,11 @@ export const getProposalUrlPath = (
   } else {
     basePath = `/proposal/${id}`;
   }
-  if (omitActiveId || (app.isCustomDomain() && !chainId)) {
```

Comment:
this logic prevented the chainID condition form hitting. Please LMK if my revision doesn't make sense. This function handles a lot of cases I think.

Reply:
I would not modify this function as it is used in many places. in `getTextRows` there is a `setRoute(threadUrl)` which works like navigate from useCommonNavigate. Check if passing third argument to setRoute as chain id would do the work?

Reply:
Agree, it is not worth risking damage to other use cases. I made a new function for this case: `getNotificationUrlPath`